packages/client/src/components/Conclusion/ConclusionOutcome.tsx

```diff
@@ -67,8 +69,19 @@ const ConclusionOutcome: FunctionComponent<ConclusionOutcomeProps> = ({
       </ComponentWrapper>
 
       {mainContent}
-
       <HideForPrint>
+        <p>
+          <a
+            href={`${
+              process.env.REACT_APP_GRAPHQL_API_URL
+            }/pdfr/${slug}?data=${encodeURIComponent(
+              JSON.stringify({ [slug]: topicData })
+            )}`}
+          >
+            Download pdf
+          </a>
+        </p>
```

Comment:
This needs to be `<Paragraph>` and `<Link>`, but I understand this is for testing purposes